backend/src/middlewares/mailSender.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { getStatusProvider } from 'db/providers/clientCourseProvider';
+import CourseStatus from 'enums/coursesEnums';
+import BadRequestError from 'classes/errors/clientErrors/BadRequestError';
+import { getUserProvider } from 'db/providers/userProvider';
+import Mail from 'classes/Mail/Mail';
+import { getCourseProvider } from 'db/providers/courseProvider';
+
+const sendMail = async (
+  req: Request,
+  res: Response<
+    never,
+    { clientCourseId: string | undefined; results: Record<'updateStatus', string> }
+  >,
+  next: NextFunction,
+) => {
+  try {
+    const { clientCourseId } = res.locals;
+    if (!clientCourseId) {
+      throw new BadRequestError('Invalid query.');
+    }
+    const { status, user: userId, course: courseId } = await getStatusProvider(clientCourseId);
+    const { email } = await getUserProvider(userId);
+    const { title } = await getCourseProvider(courseId.toString(), userId.toString());
+    if (status !== CourseStatus.approved && status !== CourseStatus.rejected) {

```

Comment:
Replace it please with an example below:
```suggestion
    if (![COURSE_STATUSES.approved, COURSE_STATUSES.rejected].includes(status)) {
```